lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -7394,13 +7394,16 @@ or the test tool
 [output_format.xml](https://github.com/galaxyproject/galaxy/blob/dev/test/functional/tools/output_format.xml)
 for more examples.
 
+For parameters that are nested in sections, conditionals, or repeats are accessed with dictionary access syntax,
+e.g. a parameter with name ``p`` that is in a conditinal with name ``c``that is in a section with name ``s``
+is referenced by ``s["c"]["p"].
```

Comment:
```suggestion
For parameters that are nested in sections, conditionals, or repeats are accessed with object access syntax,
e.g. a parameter with name ``p`` that is in a conditional with name ``c``that is in a section with name ``s``
is referenced by ``s.c.p``"].
```

Reply:
Works definitely as `s["c"]["p"]` just used it it https://github.com/galaxyproject/tools-iuc/pull/5956 and also double checked with a few print statements in the code (but not sure if `s.c.p` does not work. Code is [here](https://github.com/galaxyproject/galaxy/blob/ffc66f2c28175f99cedfd50998696fb38461f402/lib/galaxy/tools/actions/__init__.py#L1232) where `parameter_context` is a `dict`).

Reply:
Lets try https://github.com/galaxyproject/tools-iuc/pull/5956/commits/8cb7361a3a9a392e03896f8c9a57c6ca706884e6 :)

Reply:
Seems to work both. But lets go with the consensus. Thanks.